Runigram.java

```diff
@@ -133,7 +173,14 @@ public static Color[][] blend(Color[][] image1, Color[][] image2, double alpha)
 	 * of the source image.
 	 */
 	public static void morph(Color[][] source, Color[][] target, int n) {
-		//// Replace this comment with your code
+		if((target.length != source.length) || (target[0].length != source[0].length)){
+			target = scaled(target, source[0].length, source.length);
+		}
+		for (int i=0; i<=n; i++){
+		double alpha = (double) ((n-1) / (n));
+		display(blend(source, target, alpha));
+		StdDraw.pause(500);
+		}

```

Comment:
1) indentations are missing
2) (double) ((n-1)/n) will allways be 0!! It's because you doing casting after the calculation of integer divided by integer. The correct solution is (n-1)/(double)n for example.
See the difference:
4/5 => 0; double (0) => 0.0
4/5.0 => 0.8;